mobile/examples/super_resolution/android/app/src/main/java/ai/onnxruntime/example/superresolution/MainActivity.kt

```diff
@@ -60,7 +72,7 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun readInputImage(): InputStream {
-        return assets.open("test_superresolution.png")
+        return assets.open("cat_224x224.png")
```

Comment:
Actually, can we leave the name as test_superresolution.png. Then instructions to use a different image will be to copy into this filename